README.md

```diff
@@ -0,0 +1,17 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Learning Linting, HTML and CSS.
+
+
+## Built With
+
+- HTML, CSS, Javascript
+
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- Please, kindly add the licensed content as the link is broken to make your readme more professional.